api/libproject/generate_appengine_cron_yaml.rb

```diff
@@ -0,0 +1,20 @@
+# Generates the App Engine cron.yaml file for a given environment. This is meant
+# to be invoked directly via Gradle before running the GAE application.
+# For now, this picks cron_{env}.yaml if it exists, cron_default.yaml otherwise.
+#
+# Future work: it will likely be desirable to support extension rather than
+# wholesale file replacement, e.g. to alter just the schedule or exclude a
+# particular cron for a given environment.
+require 'fileutils'
+
+env = ARGV[0]
+
+cron_dir = "src/main/webapp/WEB-INF"
+to_cron_path = ->(suffix) { "#{cron_dir}/cron_#{suffix}.yaml" }
```

Comment:
I'm curious about this convention of declaring "named lambdas" and then invoking them with `.call`. I don't feel like it's a readability win, and if we need it for any kind of functional magic (passing them to algorithms and whatnot), we don't need it yet. 

Same question in Java code btw: now that we have method references, it doesn't seem like it should be necessary (unless you need captures).

Reply:
I'd thought it was needed for the closure, but apparently not.

Reply:
Oh cool. I last used Ruby ~10 years ago so I'm interested in learning more.